packages/create-sitecore-jss/src/templates/node-xmcloud-proxy/README.md

```diff
@@ -1,32 +1,41 @@
 # Node XM Cloud Proxy
 
-> Sitecore JSS Proxy for XM Cloud is considered experimental.
+> Sitecore XM Cloud Proxy is in public beta.
+> If you encounter any issues or bugs, please submit a new issue in the [Jss GitHub Repo](https://github.com/Sitecore/jss)
 
-[Documentation](TODO)
+[Documentation]((https://doc.sitecore.com/xmc/en/developers/jss/latest/jss-xmc/introducing-sitecore-javascript-rendering-sdk.html)) 
 
-This proxy will serve as the backbone for supporting various SPA frameworks by handling server-side rendering (SSR), data queries, and middleware functionalities.
+This Node.js-based proxy app is the backbone for enabling seamless integration between XM Cloud and various SPA frameworks like React, Angular, or Vue laying the groundwork for future JSS starter kits built for other front-end JavaScript frameworks. It acts as a middleware layer to handle critical functionalities such as server-side rendering (SSR), enabling editing and personalization, A/B component testing, and integrating Sitecore Forms. By serving as the rendering host, it ensures a smooth connection between Sitecore XM Cloud services and your front-end applications, making it easier to build dynamic, personalized, and localized experiences for users.
 
-This is a sample setup showing one of how you can configure XM Cloud rendering server.
+This is a sample setup showing one of how you can configure XM Cloud rendering server on top of Node.js and Express.js
 
 ## Features Supported
 
 - `Context ID`: The Context ID environment variable simplifies setting up and configuring XM Cloud solutions. It's a unified identifier that maps to all your configured resources, such as content, sites, files, forms, and integration settings.
 
 - `XM Cloud Pages editing integration`: full integration with Pages - the dynamic visual page editor of XM Cloud.
 
-- `XM Cloud proxy personalization` with embedded personalization and A/B Component Test support.
+- `XM Cloud proxy personalization` with embedded personalization and Component A/B Testing support.
 
 - `Forms support`: provides the capability to consume and post Sitecore Forms from JSS apps. Sitecore Forms is a form-authoring framework that enables marketers to author their own forms, collect data, and analyze form performance.
 
-## Configuration Setup
+- `Internationalization` support
 
-The config.ts file in this proxy app handles essential configurations for server-side rendering, data queries, and middleware functionalities. Here are the main configurations defined:
+> The following features and integrations are not supported by Node XM Cloud Proxy:
+> - Multisite
+> - XM CLoud Components application
+> - BYOC components
```

Comment:
As elsewhere, please confirm with Anastasia whether we need to say BYOC here as well as the components app.